idaes/models/properties/activity_coeff_models/tests/test_ideal_ideal_FcTP.py

```diff
@@ -69,6 +72,8 @@ def test_build(self, model):
         assert not hasattr(model.fs.state_block_ideal_vl[0], "eq_activity_coeff")
         assert not hasattr(model.fs.state_block_ideal_vl[0], "eq_mol_frac_out")
 
+        iscale.calculate_scaling_factors(model.fs.state_block_ideal_vl[0])
```

Comment:
This needs to go in the fixture, not the build test. The reason being that the build test is marked as a unit test, but the tests involving solvers are marked as component tests; thus if you try to run only the component tests, this line will not be executed and the solver tests will be missing scaling.